app/views/videos/show.html.haml

```diff
@@ -7,6 +7,37 @@
         %header
           %h3= @video.title
           %h4=link_to @video.category.name, category_path(@video.category)
-          %span Rating: 4.5/5.0
+          %span
+            - if @video.average_rating != nil
+              = "Rating: #{@video.average_rating}/5.0"
```

Comment:
This `= "Rating: #{@video.average_rating}/5.0"` should change to:

`Rating: #{@video.average_rating}/5.0`